Osiris/Hooks.h

```diff
@@ -55,6 +55,8 @@ class Hooks {
     HookType sound;
     HookType surface;
     HookType viewRender;
+    VmtSwap networkChannel;
```

Comment:
change to hooktype, vmt is probably detected and there is no reason not to use hooktype

Reply:
i wish i can lol, codes pasted

Reply:
any1 wanna do it for me?